src/bundle/Resources/public/js/scripts/helpers/timezone.helper.js

```diff
@@ -1,29 +1,77 @@
 (function(global, doc, eZ, moment) {
     const userPreferedTimezone = eZ.adminUiConfig.timezone;
-    const userPreferedFullDateFormat = eZ.adminUiConfig.dateFormat.full;
-    const userPreferedShortDateFormat = eZ.adminUiConfig.dateFormat.short;
+    const userPreferedFullDateTimeFormat = eZ.adminUiConfig.dateFormat.full_datetime;
+    const userPreferedFullDateFormat = eZ.adminUiConfig.dateFormat.full_date;
+    const userPreferedFullTimeFormat = eZ.adminUiConfig.dateFormat.full_time;
+    const userPreferedShortDateTimeFormat = eZ.adminUiConfig.dateFormat.short_datetime;
+    const userPreferedShortDateFormat = eZ.adminUiConfig.dateFormat.short_date;
+    const userPreferedShortTimeFormat = eZ.adminUiConfig.dateFormat.short_time;
 
     const convertDateToTimezone = (date, timezone = userPreferedTimezone) => {
         return moment(date).tz(timezone);
     };
-    const formatDate = (date, format = userPreferedFullDateFormat) => {
+    const formatDate = (date, format, timezone = null) => {
+        if (timezone) {
+            date = convertDateToTimezone(date, timezone);
+        }
+
         return moment(date).formatICU(format);
     };
-    const formatDateWithTimezone = (date, timezone = userPreferedTimezone, format = userPreferedFullDateFormat) => {
-        return formatDate(convertDateToTimezone(date, timezone), format);
+    const formatFullDateTime = (date, timezone = null, format = userPreferedFullDateTimeFormat) => {
+        return formatDate(date, format, timezone);
+    };
+    const formatFullDate = (date, timezone = null, format = userPreferedFullDateFormat) => {
```

Comment:
What is the reason for having this method? It's not used anywhere, there are more methods like this here.

Reply:
Those function will be added to `helpers.timezone` in 2.0 , currently it is not possible because of a name conflict with deprecated function `formatShortDate`

Reply:
So this is the dead code? No point of having this in LTS 2.5.